phyui/plot/waveforms.py

```diff
@@ -325,6 +326,7 @@ class WaveformView(PanZoomCanvas):
     def __init__(self, **kwargs):
         super(WaveformView, self).__init__(**kwargs)
         self.visual = Waveforms()
+        self.resizable = Bool(defaultValue=True, sync=True)
```

Comment:
yes exactly... I wonder how it worked then...

On Mon, Mar 2, 2015, 17:37 Cyrille Rossant <notifications@github.com> wrote:

> In phyui/plot/waveforms.py
> <https://github.com/kwikteam/phy-ui/pull/1#discussion_r25612836>:
>
> > @@ -325,6 +326,7 @@ class WaveformView(PanZoomCanvas):
> >      def __init__(self, **kwargs):
> >          super(WaveformView, self).__init__(**kwargs)
> >          self.visual = Waveforms()
> > +        self.resizable = Bool(defaultValue=True, sync=True)
>
>  IPython/utils/traitlets.py
>
> you mean this
> <https://github.com/ipython/ipython/blob/master/IPython/utils/traitlets.py#L321>
> ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kwikteam/phy-ui/pull/1/files#r25612836>.
>